tsconfig.json

```diff
@@ -1,26 +1,8 @@
 {
+  "extends": "@edx/typescript-config",
   "compilerOptions": {
-    "allowJs": true,
-    "allowSyntheticDefaultImports": true,
-    "declaration": true,
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "jsx": "react",
-    "lib": ["dom", "es6", "dom.iterable"],
-    "isolatedModules": true,
-    "module": "ES6",
-    "moduleResolution": "node",
-    "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": false,
-    "noImplicitThis": true,
-    "noUnusedParameters": true,
-    "skipLibCheck": true,
-    "sourceMap": true,
-    "strict": true,
-    "strictFunctionTypes": false,
-    "target": "ES6",
     "rootDir": ".",
-    "outDir": "dist"
+    "outDir": "dist",
```

Comment:
nit: does JSON support trailing commas on the final property defined in object? it doesn't seem my IDE is complaining about the trailing comma, but I don't think it's technically valid JSON 🤷 

<img width="399" alt="image" src="https://github.com/user-attachments/assets/bb00c0cd-52cc-4b30-b92a-bd3c258cc342" />